docs/index.md

```diff
@@ -4,7 +4,141 @@ hide:
+Accessibility
+</h1>
+
+There is an elevator available to reach the venue. The bathrooms are wheelchair accessible.
+
+---
+
+<h1 align="left" style="margin-top: 0px;">
+Registration form
+</h1>
+
+Please register your self at <a href="https://cfp.3mdeb.com/hackathon-2022/submit/" target="_blank">https://cfp.3mdeb.com/hackathon-2022/</a>
+
+---
+<h1 align="left" style="margin-top: 0px;">
+Code of Conduct
+</h1>
+
+In the interest of fostering an open and welcoming environment, we as organizers
+and maintainers pledge to make the event and our community a harassment-free
+experience for everyone, regardless of age, body size, disability, ethnicity,
+gender identity and expression, level of experience, nationality, personal
+appearance, race, religion, sexual identity and orientation, or other
+characteristic.
+
+
+Examples of behavior that contributes to creating a positive environment include:
+
+*  Using welcoming and inclusive language
+*  Being respectful of differing viewpoints and experiences
+*  Gracefully accepting constructive criticism
+*  Focusing on what is best for the community
+*  Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+*  The use of sexualized language or imagery and unwelcome sexual attention
+or advances
+*  Trolling, insulting/derogatory comments, and personal or political attacks
+*  Reinforcing stereotypical models for illustration of non-technical users
+(e.g. our mothers/grandmothers, etc.)
+*  Public or private harassment, as defined by the Citizen Code of Conduct
+*  Publishing others’ private information, such as a physical or electronic
+address, without explicit permission
+*  Other conduct which could reasonably be considered inappropriate in a
+professional setting
+
+---
+
+<h1 align="left" style="margin-top: 0px;">
+Hackathon Schedule
+</h1>
+
+The full schedule is available at: <a href="https://cfp.3mdeb.com/hackathon-2022/schedule/" target="_blank">https://cfp.3mdeb.com/hackathon-2022/schedule/</a>
+
+---
+
+<h1 align="left" style="margin-top: 0px;">
+Recording
+</h1>
+
+The live presentation will be recorded, as well as some of the highlights of
+the event. By joining the event, you consent to the recording of your image for
+marketing purposes.
+
+---
+<h1 align="Left" style="margin-top: 0px;">
+Social media
+</h1>
+<h2 align="Left" style="margin-top: 0px;">
+  <a href="https://twitter.com/3mdeb_com" class="fa fa-twitter" target="_blank"></a>
+  <a href="https://twitter.com/3mdeb_com" target="_blank"> 3mdeb </a>
+</h2>
+<h2 align="Left" style="margin-top: 0px;">
+  <a href="https://www.reddit.com/user/3mdeb/" class="fa fa-reddit" target="_blank"></a>
+  <a href="https://www.reddit.com/user/3mdeb/" target="_blank"> 3mdeb </a>
+</h2>
+
+---
+
+<h1 align="Left" style="margin-top: 0px;">
+Sponsors
+
+Want to support the Hackathon 2022?
+  <a href="https://calendly.com/3mdeb/" target="_blank">Let's talk about it!</a>
```

Comment:
We probably should mention Health and Safety compliance to Polish law with some link to public information about that.

Reply:
I created new section.